### ISSUE TITLE
Ethical Concern

### ISSUE THREAD

**Issue by zundaxu** [uchicago-computation-workshop/chris_bail#9]
Thank you so much for presenting this paper and it's really impressive!

However, I still want to raise some ethical concern about this study. Although you have mentioned that the research and the deception in this study had been approved by IRB, I am still wondering whether there exist ethical issues, since after all, the manipulation was conducted on the real life and real ideology, and the results of polarized or less polarized is resulted by your manipulation rather than by nature. Is it really ethical to manipulate others life? Can you please give more explanation on this issue?

Thanks a lot !

**Comment by tamos** [comment]
Good point - But these non-conforming views are in the public domain; in fact these people probably encounter these ideas in their daily life (news, etc.)
